### PR TITLE
Draw entities

### PR DESCRIPTION
**Pull request by eitanfr**
refactor to use cesium entities api

### CODE REVIEW

File: src/services/drawers/polyline-drawer/polyline-drawer.service.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ *  This drawer is responsible of drawing polylines.
+ */
+@Injectable()
+export class PolylineDrawerService extends PrimitivesDrawerService {
+	constructor(cesiumService: CesiumService) {
+		super(Cesium.PolylineCollection, cesiumService);
+	}
+
+  add(cesiumProps: any) {
+    if (cesiumProps.color) {
```
Comment (eitanfr) [inline src/services/drawers/polyline-drawer/polyline-drawer.service.ts]: 1. why polyline primitive and not entiy?
2. sure we need this color setting?

File: demo/app/components/tracks-layer/tracks-layer.component.html [hunk 2]
```diff
@@ -1,35 +1,49 @@
...
+
+    <ac-polyline-desc *ngIf="showVelocityVectors()" props="{
+        positions: [track.position, track.futurePosition],
+        width: 1,
+        color: getPolylineColor(),
+        constantColor: true
+    }">
+    </ac-polyline-desc>
+
+    <ac-ellipse-desc *ngIf="showEllipses()" props="{
```
Comment (tomermoshe) [inline demo/app/components/tracks-layer/tracks-layer.component.html]: done

File: src/services/basic-desc/basic-desc.service.ts [hunk 3]
```diff
@@ -1,69 +1,69 @@
...
 export class BasicDesc implements OnInit, OnDestroy {
-	@Input()
-	props: any;
+  @Input()
+  props: any;
 
-	protected _primitiveMap = new Map();
-	private _propsEvaluateFn: Function;
-	private _propsAssignerFn: Function;
+  protected _mapEntitiesMap = new Map();
```
Comment (tomermoshe) [inline src/services/basic-desc/basic-desc.service.ts]: fixed

File: src/services/basic-desc/basic-desc.service.ts [hunk 4]
```diff
@@ -1,69 +1,69 @@
...
-		return this._propsEvaluateFn(this._computationCache, context);
-	}
+  protected _propsEvaluator(context: Object): any {
+    return this._propsEvaluateFn(this._computationCache, context);
+  }
 
-	protected _getPropsAssigner(): (primitive: Object, desc: Object) => Object {
-		return (primitive: Object, desc: Object) => this._propsAssignerFn(primitive, desc);
-	}
+  protected _getPropsAssigner(): (primitive: Object, desc: Object) => Object {
```
Comment (tomermoshe) [inline src/services/basic-desc/basic-desc.service.ts]: done

**Pull request merged**